src/scenario.jl

```diff
@@ -90,7 +90,7 @@ function run_scenarios(
 
     # Cross product between rcps and param_df to have every row of param_df for each rcp
     rcps_df = DataFrame(; RCP=parse.(Int64, RCP))
-    scenarios_df = crossjoin(scens, rcps_df)
+    scenarios_df = crossjoin(scens[:, Not(:RCP)], rcps_df)
```

Comment:
Apparently this is raising an error when I run the model

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/41bd11fe-6109-4ef6-85fe-ee4435b62827)

Reply:
:facepalm: let me check what's happened there...

Reply:
Okay, I don't know what's happening as it works for me?

But I know which commit is causing it.

Reply:
Could you pull and try again? Issues should be resolved.

Reply:
hum.. now I can `run_model` but I got an error when running the tests (also, it's taking too long to run the tests, I think it was more than 10 mins...that's insane)

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/5bddeac8-7e63-42e4-8957-42e1a1170815)

Reply:
Ah yes Aqua tests will fail as I had to comment out the compat entries.

Strange that it takes 10mins though.

Reply:
Just to clarify, the idea is to leave is this way for now or you are going to fix that in this PR?

Reply:
So I ran it, the tests failed (but we can comment the compat tests out) but it took way less than 10mins.

Reply:
> So I ran it, the tests failed (but we can comment the compat tests out) but it took way less than 10mins.

Hum... I don't know what happened here... 🤔 

I think it's a good idea to comment the compat tests out and open an issue to fix them later. :)

Reply:
I'll do that in an hour or so

Reply:
All done. Tests complete in well under <4mins for me (didn't do a comprehensive timing but based on output)

```julia-repl
Test Summary: | Pass  Total   Time
Aqua          |    4      4  18.8s
Running 64 scenarios for RCP 45 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| Time: 0:01:03        
Test Summary:       | Pass  Total  Time
Temporal clustering |   35     35  0.0s
Test Summary:  | Pass  Total  Time
Domain loading |    4      4  3.8s
Test Summary:        | Pass  Total  Time
Connectivity loading |    3      3  0.3s
Test Summary:      | Pass  Total  Time
Environmental data |    2      2  0.2s
Test Summary:  | Pass  Total  Time
Initial covers |    1      1  0.0s
Test Summary:          | Pass  Total  Time
Cyclone mortality data |    2      2  0.2s
Test Summary:  | Pass  Total   Time
Domain loading |    7      7  32.2s
Test Summary:         | Pass  Total  Time
Truncated normal mean |    2      2  0.0s
Test Summary:        | Pass  Total  Time
Truncated normal cdf |    2      2  0.0s
Test Summary:           | Pass  Total  Time
proportional adjustment |   10     10  1.2s
Test Summary: | Pass  Total  Time
Coral Spec    |   27     27  0.3s
Test Summary: | Pass  Total  Time
Fecundity     |    2      2  0.3s
Test Summary:     | Pass  Total  Time
Larval Production |    1      1  0.8s
Test Summary: | Pass  Total  Time
Recruitment   |    7      7  0.0s
Test Summary: | Pass  Total  Time
growth model  |    6      6  0.4s
Test Summary: | Pass  Total  Time
NetCDF        |    5      5  0.5s
Test Summary: | Pass  Total  Time
axes_names    |    2      2  0.2s
Test Summary: | Pass  Total  Time
sort_axis     |    2      2  0.7s
Test Summary: | Pass  Total  Time
ZeroDataCube  |   12     12  0.6s
Test Summary: | Pass  Total  Time
DataCube      |   21     21  0.9s
Test Summary:           | Pass  Total  Time
Relative Shelter Volume |    6      6  5.8s
Test Summary: | Pass  Total  Time
sample        |   33     33  2.1s
Test Summary:     | Pass  Total  Time
Targeted sampling |   11     11  5.3s
Test Summary: | Pass  Total     Time
Sample bounds |  565    565  1m46.5s
Test Summary: | Pass  Total  Time
Seeding       |   75     75  3.0s
Test Summary:                   | Pass  Total  Time
Extracting component parameters |    4      4  0.5s
Test Summary:                  | Pass  Total  Time
Validate included MCDA methods |    1      1  5.0s
Test Summary: | Pass  Total  Time
text_display  |   28     28  0.0s
Test Summary:    | Pass  Total  Time
Full example run |    2      2  7.8s
     Testing ADRIA tests passed
```

Gotta do something about these excessive warnings that spam the terminal though:

```
┌ Warning: Can not determine size of element type. Using DiskArrays.fallback_element_size[] = 100 bytes
└ @ DiskArrays C:\Users\tiwanaga\.julia\packages\DiskArrays\MpOpv\src\chunks.jl:321
```